docker-compose.yml

```diff
@@ -153,6 +153,22 @@ services:
     volumes:
     - ./web/dkim/keys:/etc/opendkim/keys
 
+  telegraf:
+    image: telegraf
+    container_name: telegraf
+    environment:
+      TELEGRAF_NAME: ${TELEGRAF_NAME}
+      TELEGRAF_INFLUXDB_DB: ${TELEGRAF_INFLUXDB_DB:?TELEGRAF_INFLUXDB_DB unset}
+      TELEGRAF_INFLUXDB_URL: ${TELEGRAF_INFLUXDB_URL:?TELEGRAF_INFLUXDB_URL unset}
+      TELEGRAF_INFLUXDB_USER: ${TELEGRAF_INFLUXDB_USER:?TELEGRAF_INFLUXDB_USER unset}
+      TELEGRAF_INFLUXDB_PASS: ${TELEGRAF_INFLUXDB_PASS:? TELEGRAF_INFLUXDB_PASS unset}
+    volumes:
```

Comment:
This looks like it will not be monitoring the host because the system directories arent mounted (https://hub.docker.com/_/telegraf?tab=description "Monitoring the Docker Engine Host")

Reply:
looks like i need to add
```
-H "HOST": "localhost"
```
to my curl query to get it to work.

Reply:
I still dont see the host system directories mounted